### PR TITLE
chore(test): create playwground e2e test

### PR DESCRIPTION
**Pull request by cbr7**
### What does this PR do?
Create e2e tests for AI lab playground creation and deletion

### What issues does this PR fix or reference?
https://github.com/containers/podman-desktop-extension-ai-lab/issues/2101

### CODE REVIEW

**Comment by cbr7** [comment]
Started a test run on windows cicd [here](https://github.com/containers/podman-desktop-extension-ai-lab/actions/runs/12389544446).

File: tests/playwright/src/ai-lab-extension.spec.ts [hunk 1]
```diff
@@ -204,6 +205,77 @@ test.describe.serial(`AI Lab extension installation and verification`, { tag: '@
...
   });
 
+  ['instructlab/granite-7b-lab-GGUF'].forEach(modelName => {
+    test.describe.serial(`AI Lab playground creation and deletion`, () => {
+      let catalogPage: AILabCatalogPage;
+      let playgroundsPage: AILabPlaygroundsPage;
+
+      const playgroundName = 'test playground';
+
+      test.skip(isLinux, `Skipping model service creation on Linux`);
```
Comment (odockal) [inline tests/playwright/src/ai-lab-extension.spec.ts:215]: just a typo after copying the message - model service.

**Comment by odockal** [comment]
follow up: if some steps were not performed, like checking the running service, sending input, etc. please create follow up issue for this.

File: tests/playwright/src/ai-lab-extension.spec.ts (hunk 1 above)
Comment (cbr7) [reply tests/playwright/src/ai-lab-extension.spec.ts]: fixed

**Pull request merged**